AEXS/aex-2.md

```diff
@@ -101,32 +105,34 @@ The wallet can also let the SDK know of extra funtionality that it provides duri
 
 2. The SDK listens for `registerProvider` and on receiving a request generates a pub-priv key pair and broadcasts a `registrationComplete` message that includes `sdk` public key that should be used by wallet provider for all the future communications.
    Message:
+
    ```json
    {
      "registrationComplete": {
-       "provider": "<DH public key>",
-       "sdk": "<DH public key>"
+        "provider": "<DH public key>",
+        "sdk": "<DH public key>"
      }
    }
    ```
+
 3. At this step, the user should be prompted by the extension/wallet to accept the incoming SDK. If the user agrees, the wallet posts the `walletDetail` message encrypted using the secret key generated (wallet private key + sdk public key) and nonce.
```

Comment:
It would help readability if you give the keys generated labels

Reply:
need more explanation or example.